packages/twenty-server/src/engine/core-modules/billing/utils/billing-webhook-entitlement-transformer.util.ts

```diff
@@ -0,0 +1,22 @@
+import Stripe from 'stripe';
+
+import { BillingEntitlementKey } from 'src/engine/core-modules/billing/enums/billing-entitlement-key.enum';
+
+export const transformStripeEntitlementEventToEntitlementRepositoryData = (
```

Comment:
file name does not meet function name

Reply:
based on data type, looks the name should reflect the fact this is an UpdatedEvent